tools/scoring/README.md

```diff
@@ -1,6 +1,12 @@
 # Scoring
 
-The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various _dimensions_.
+The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various [_dimensions_](#dimensions). Broadly, its process is thus:
+
+1. Filter out irrelevant entities (e.g. those in the solution which have a noncanonical type)
+2. Create pairs of entities from the _proposed_ and _solution_ lists by matching `cloud_device_id` (for reporting devices) or raw field names (for virtual devices)
```

Comment:
can you please add a reference to the cloud device id: https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md#config-format

Reply:
(Will address in https://github.com/google/digitalbuildings/pull/549)

Reply:
https://github.com/google/digitalbuildings/pull/549/commits/b97384fb499100da6f85080782d2fcd1bf89d237